lib/sshutils/sftp/sftp.go

```diff
@@ -358,7 +358,7 @@ func expandPath(pathStr string) (string, error) {
 		return ".", nil
 	}
 	if pfxLen == 1 && len(pathStr) > 1 {
-		return "", trace.BadParameter("expanding remote ~user paths is not supported, specify an absolute path instead")
+		return "", trace.BadParameter("expanding remote ~user paths is not supported, specify an absolute path instead of %q", pathStr)
```

Comment:
i could see someone innocently changing this error message, not realizing we check for it on the client side. Should we use a constant or at least leave a comment referencing that this specific error string is checked?